### PR TITLE
New package: aspell-it-2.2_20050523-0

### PR DESCRIPTION
**Pull request by anddam**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibx)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - armv7l (crossbuild)

### CODE REVIEW

File: srcpkgs/aspell-it/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'aspell-it'
+pkgname=aspell-it
+version=2.2+20050523~0
```
Comment (sgn) [inline srcpkgs/aspell-it/template:3]: Debian has a newer one (with prettier version scheme) https://packages.debian.org/bullseye/aspell-it

Comment (anddam) [reply srcpkgs/aspell-it/template:3]: Fine by me, I went with what I thought was the ["official" list](https://ftp.gnu.org/gnu/aspell/dict/0index.html).

Comment (anddam) [reply srcpkgs/aspell-it/template]: @sgn was I supposed to resolve the conversation or were you?

File: srcpkgs/aspell-it/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'aspell-it'
+pkgname=aspell-it
+version=2.4
+revision=1
+_version="${version}-20070901-0"
```
Comment (sgn) [inline srcpkgs/aspell-it/template:5]: The commit message is not correct anymore.

Comment (anddam) [reply srcpkgs/aspell-it/template:5]: Now it is again.

**Pull request merged**